h2o-core/src/main/java/water/fvec/Frame.java

```diff
@@ -885,7 +889,7 @@ void preparePartialFrame( String[] names ) {
     if( _keys != null ) delete_and_lock();
     else write_lock();
     _keys = new Vec.VectorGroup().addVecs(names.length);
-    setNames(names);
+    setNamesNoCheck(names);
```

Comment:
Vlad:  One question for you.  The reason to use SetNamesNoCheck at preparePartialFrame in Frame.java is because the frame building is not complete.  It can still grow either in length or in height?  Let me know.  Wendy